posix.mak

```diff
@@ -722,7 +722,7 @@ $G/docs-latest.json : ${DMD_LATEST} ${DMD_LATEST_DIR} \
 	find ${DMD_LATEST_DIR}/src -name '*.d' > $G/.latest-files.txt
 	find ${DRUNTIME_LATEST_DIR}/src -name '*.d' | \
 		sed -e /unittest.d/d -e /gcstub/d >> $G/.latest-files.txt
-	find ${PHOBOS_LATEST_DIR} -name '*.d' -not -path '${PHOBOS_LATEST_DIR}/generated/*' | \
+	find ${PHOBOS_LATEST_DIR} -name '*.d' | grep -E "(etc|std)" | \
```

Comment:
How about `find ${PHOBOS_LATEST_DIR}/std ${PHOBOS_LATEST_DIR}/etc -name '*.d'`?

(You can specify multiple dirs to `find` before specify expressions, such `-name` - full syntax:
`find [-H] [-L] [-P] [-D debugopts] [-Olevel] [path...] [expression]`.)

Reply:
Sure. Changed.